examples/streaming_compression.c

```diff
@@ -15,9 +15,12 @@
 #include <zstd.h>      // presumes zstd library is installed
 #include "common.h"    // Helper functions, CHECK(), and CHECK_ZSTD()
 
+#define THREADS 4
```

Comment:
I don't think a macro is necessary here. You can pass a `const int nbThreads = 4` from `main()` and add an `nbThreads` arg to `compressFile_orDie`. That way it's more consistent with the change to `cLevel` getting passed from main as well (and in fact, you could add `nbThreads` as an additional arg to the CLI too).